aries-site/src/examples/templates/forms/FilterExample.js

```diff
@@ -105,6 +106,11 @@ export const FilterExample = () => {
               <CheckBoxGroup options={sellers} id="sellers" name="sellers" />
             </FormField>
           )}
+          <Box direction="row-responsive" gap="medium" pad={{ top: 'medium' }}>
```

Comment:
Can we check on the responsive behavior here? Let's make sure it is aligned with the direction which @jeffstolz is heading. I believe the direction is calling for only primary buttons to occupy full-width. Let's also consider whether this scenario is covered in that guidance.

![Screen Shot 2021-12-03 at 9 41 39 AM](https://user-images.githubusercontent.com/1756948/144639795-4356d7ba-a5c8-4dab-bcef-d5f37a4a6565.png)

Reply:
Oh got it, I must have missed the primary only. I will take a look at his work there.

Reply:
Followed up with Jeff -- he is going to revise his guidance to clarify, but he thinks that on mobile buttons should go full-width in these contexts and color/styling provides correct indication of hierarchy.